New getting started for Angular (#2204)
aws-amplify/docs: docs/start/getting-started/fragments/angular/setup.md

## Create a new Angular app

> <inline-fragment src="~/ui/auth/fragments/angular/new-angular-app.md"></inline-fragment>

Reviewer: Why was this fragment removed for the below content?

```bash
npm install -g @angular/cli
ng new amplify-app
cd amplify-app
```

### Angular 6+ Support

> Currently, the newest versions of Angular (6+) do not include shims for 'global' or 'process' which were provided in previous versions. Add the following to your `polyfills.ts` file to recreate them: 

Reviewer 2: ```suggestion
Currently, the newest versions of Angular (6+) do not include shims for 'global' or 'process' as provided in previous versions. Add the following to your `polyfills.ts` file to recreate them: 
```

amplify init
```

> When you initialize Amplify you'll be prompted for some information about the app.  For newer versions of Angular, you will have to change the Distribution Directory Path from `dist` to `dist/amplifyapp` to match how Angular will build your project.

Reviewer 2: "newer versions" -- if possible, get specific. (6+) ?

Author: This is intended as a possible troubleshooting. Normally this won't be necessary.

Inside the app directory, install the Amplify Angular library and run your app:

```bash
npm install --save @aws-amplify/auth @aws-amplify/api @aws-amplify/pubsub @aws-amplify/storage @aws-amplify/ui-angular
ng serve
```

The `@aws-amplify/ui-angular` package is a set of Angular components and an Angular provider which helps integrate your application with the AWS-Amplify library.  It supports Angular 5.0 and above.  It also includes a [supplemental module](#ionic-4-components) for Ionic-specific components.

> <amplify-callout>Angular CLI output warnings: if you see CommonJS or AMD dependencies optimization bailouts warnings using Angular +9 you can use this [gist](https://gist.github.com/gsans/8982c126c4fef668c094ff288f04241b) to remove them. More details about these [here](https://angular.io/guide/build#configuring-commonjs-dependencies).

Reviewer 2: Instead of speaking in generalities, are there salient excerpts we can extract from the error messages?

e.g.,

"If you get an `TechnicalReferenceFromOutput` warning while using Angular 9+, \[see these notes\](http://notes.link)".

Author: There's no specific error. Output is actually worst: 
```
WARNING in /Users/ehanchett/programming/work/eig/front/crty-ng-ui/node_modules/@aws-sdk/client-s3/dist/es/runtimeConfig.browser.js depends on @aws-sdk/eventstream-serde-browser. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```
